libs/SalesforceSDKCore/SalesforceSDKCore/SFUserAccountManagerNotificationsTests.m

```diff
@@ -0,0 +1,512 @@
+/*
+ Copyright (c) 2017-present, salesforce.com, inc. All rights reserved.
```

Comment:
@bhariharan How does Xcode even do this? Logically it still shows under tests group. I'll move it. Thanks for catching it.